OneCommunityGlobal/HighestGoodNetworkApp
# Shereen purchase equipment

# Description
Adds functionality for the user to place purchase order for equipment. [/bmdashboard/equipment/purchase/] (http://localhost:3000/bmdashboard/equipment/purchase/).

Implements # (WBS) 9.2.1

![Screenshot 2024-05-09 105828](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/7041610/7990c0c1-8e6b-4c76-8f95-528e1db47761)

## Related PRS (if any):
This frontend PR is related to the [#946 backend PR. https://github.com/OneCommunityGlobal/HGNRest/pull/946](https://github.com/OneCommunityGlobal/HGNRest/pull/946)
To test this backend PR you need to check out the [#946 frontend PR.](https://github.com/OneCommunityGlobal/HGNRest/pull/946)
…

## Main changes explained:
- A new route was added to routes.js to correspond to "/bmdashboard/equipment/purchase"
- Created PurchaseEquipment.jsx for the component.

## How to test:
1. check into current branch
2. Run commands npm install and npm run start:local to launch the front-end application
3. Clear site data/cache
4. Log in as any user
5. Go to /bmdashboard (http://localhost:3000/bmdashboard) and log in again.
5. Navigate to [/bmdashboard/equipment/purchase] (http://localhost:3000/bmdashboard/equipment/purchase/)
6. Place a purchase order for an equipment.
7. Verify that the green success message has been displayed after the form submission.

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/7041610/3ab86dee-95a8-4501-a15d-39d1d254f861
